src/inputs/Switch.tsx

```diff
@@ -13,6 +14,8 @@ export interface SwitchProps {
   disabled?: boolean;
   /** Input label */
   label: string;
+  /** Where to put the label. */
+  labelStyle?: "form" | "inline";
```

Comment:
The current switch had labels off to the right, but when in a form with other fields, I think it's more consistent to put the label on top.